notifier.go

```diff
@@ -267,6 +268,37 @@ func buildCreateNoticeURL(host string, projectId int64) string {
 	return fmt.Sprintf("%s/api/v3/projects/%d/notices", host, projectId)
 }
 
+func NewBlacklistKeysFilter(keys ...string) func(*Notice) *Notice {
```

Comment:
Change `keys ...string` to `keys ...interface{}` so it can accept string or regexp like this:

```
gobrake.NewBlacklistKeysFilter("password", regexp.MustCompile("(?i)(user)"))
```